src/screens/CreateWalletScreen.tsx

```diff
@@ -79,35 +79,53 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
         {i18n.notifications.receiveTransactionDescription}
         <Text style={styles.boldedText}>{this.props.email}</Text>
       </Text>
+      <Text style={[styles.notificationDescription, styles.note]}>
+        <Text style={styles.boldedText}>{i18n.notifications.noteFirst}</Text>
+        {i18n.notifications.noteSecond}
+      </Text>
     </>
   );
 
+  navigateToSuccesfullNotificationSubscriptionMessage = () => {
```

Comment:
I like it 👍 we should more often extract logic to functions with meaningful names

Reply:
...but I'm wondering if it'd be better to place such functions inside functions? I mean, if we have a method invoked on press of the submit button, wouldn't it better to have there:
```
handleOnPress() { 
  const showSuccessMessage = () => { (...) }

  const showFailureMessage = () => { (...) }

  (...)
  
  doSomething({
    onSuccess: showSuccessMessage
    onFailure: showFailureMessage
  });
}
```  

instead:
```
showSuccessMessage() { (...) }

showFailureMessage() { (...) }

handleOnPress() { 
  doSomething({
    onSuccess: showSuccessMessage
    onFailure: showFailureMessage
  });
}
```

maybe it's a small difference, but this code is more clear IMHO - methods of the components should describe behaviour of the component, but functions `showSuccessMessage`  `showFailureMessage` are just helpers for `handleOnPress` in this particular case.

Hopefully, I got my point

Reply:
I like the second version more, the problem with the first version is that the function handleOnPress will still be very long which in my opinion would decrease readability